test/wasi/README.md

```diff
@@ -0,0 +1,5 @@
+# WASI Tests
+
+Compile with `clang c/x.c -target wasm32-unknown-wasi -o wasm/x.wasm`
+
+The clang version used must be built with wasi-sysroot.
```

Comment:
\o/ \o/  Really excited this is testing and running vanilla clang/wasi-sysroot output.